Article/Article.js

```diff
@@ -109,6 +109,92 @@ const data = [
 
   Step 4: Map over the data, creating a component for each oject and add each component to the DOM as children of the 'articles' div.
 
-  Step 5: Add a new article to the array. Make sure it is in the same format as the others. Refresh the page to see the new artible
+  Step 5: Add a new article to the array. Make sure it is in the same format as the others. Refresh the page to see the new article.
 
 */
+
+
+
+
+function thirdParagraphCreator(thirdParagraph){
```

Comment:
great job with making different element creator functions.  Is there a way you can think of to make it more generic so that you can just pass it different data and have it make different elements?